### DISCUSSION TITLE
Feature Request:  Audio Output Control

### DISCUSSION THREAD

**Discussion by spackletoe** [baffler/Transparent-Twitch-Chat-Overlay#79] [Ideas]
The sound effect when a new chat comes in is one that I've found to be rare in this type of app, but it's a big benefit for those of us that don't have a ton of traffic in our chats.  I've hade trouble routing the audio for this app to a private channel that can only be heard by me and not that goes out to my stream.  I use El Gato's WaveLink software and I can usually route apps' audio wherever I want by selecting app audio output within Windows Sound Settings, but this one refuses top be routed.  Can a "audio output" selection be added to the app so there's no question where it is to direct audio?

Thanks, I love this app!

![image](https://user-images.githubusercontent.com/132308/111556570-269ab400-8748-11eb-9ee4-33d261eb0e2f.png)
(even with tis set, the audio still comes through the "System" channel.)

**Comment by baffler** [comment]
Yeah it's on the todo list. I plan on changing on the way the app will play audio and use something that will let you change the audio output.

**Reply by spackletoe** [reply]
Thanks for the quick reply.  I, and my viewers, are looking forward to that release!

**Reply by curlott** [reply]
for now:
use Wave Link Aux 1 as ur default audio device, run Twitch Chat Overlay, switch back to your other default audio device. Sounds should still route through Wave Link Aux 1. At least for me.

**Reply by Retronomia** [reply]
Hello! I was wondering if there are still any plans to make it so one can change the audio sounds to be on a device other than default (this thread is ~2 years old and I honestly don't know how people view necroposting on github). I really love the notification sounds, but don't want viewers to be hearing them as well. Thanks!

**Reply by spackletoe** [reply]
Hey @Retronomia! There wasn’t an update, but until there is, I found another app that is pretty robust and lets me do what you’re after. https://chatty.github.io/

**Reply by Retronomia** [reply]
I prefer the UI of TTCC and curlott's workaround is good enough for the time being, but thanks

For some reason twitch chat software isn't very searchable on google which is odd since a couple of options are actively trying to monetize

**Comment by spackletoe** [comment]
That's probably how I got it to work once,, and I had no idea how to get it that way again.  I'll give it a shot.  Thanks!

**Comment by baffler** [comment]
Finally, this feature is added with [v0.9.5](https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/tag/v0.9.5)!
